tests/commands/network.py

```diff
@@ -357,6 +357,31 @@ def runTest(self):
         self.assert_parse(vlan_over_bond_cmd)
         self.assert_parse(vlan_over_bond_cmd + " --bindto mac")
 
+class F38_TestCase(F27_TestCase):
+    def runTest(self):
+        F27_TestCase.runTest(self)
+
+        # dns search
+        network_data = self.assert_parse("network --device eth0 --ipv4-dns-search example.com --ipv6-dns-search example.com")
+        self.assertEqual(network_data.ipv4_dns_search, "example.com")
+        self.assertEqual(network_data.ipv6_dns_search, "example.com")
+        network_data = self.assert_parse("network --device eth0")
+        self.assertIsNone(network_data.ipv4_dns_search)
+        self.assertIsNone(network_data.ipv6_dns_search)
+        self.assert_parse_error("network --ipv4-dns-search example.com --ipv6-dns-search example.com")
+
+        # ignore auto dns
+        network_data = self.assert_parse("network --device eth0 --ipv4-ignore-auto-dns yes --ipv6-ignore-auto-dns yes")
```

Comment:
I don't think having yes/no/true/false available really adds anything. The option itself is verbose enough to be understood.